x-pack/legacy/plugins/uptime/public/components/monitor/monitor_duration/monitor_duration.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiTitle } from '@elastic/eui';
+import { LocationDurationLine } from '../../../../common/types';
+import { MLIntegrationComponent } from '../ml/ml_integeration';
+import { AnomalyRecords } from '../../../state/actions';
+import { DurationChartComponent } from '../../common/charts';
+
+interface DurationChartProps {
+  loading: boolean;
+  hasMLJob: boolean;
+  anomalies: AnomalyRecords | null;
+  locationDurationLines: LocationDurationLine[];
+}
+
+/**
+ * This chart is intended to visualize monitor duration performance over time to
+ * the users in a helpful way. Its x-axis is based on a timeseries, the y-axis is in
+ * milliseconds.
+ * @param props The props required for this component to render properly
+ */
+export const MonitorDurationComponent = ({
+  locationDurationLines,
+  anomalies,
+  loading,
+  hasMLJob,
+}: DurationChartProps) => {
+  return (
+    <>
```

Comment:
Do we need this fragment? Kind of a nit but I see these from time to time having only one child; doesn't have an impact on output, but it's probably cleaner to remove them when not needed.

Reply:
removed it